src/ui/Card.tsx

```diff
@@ -170,15 +174,29 @@ export function Card(props: CardProps): JSX.Element {
   const [clock] = useState(new Clock());
   const panDrawX = useAnimation(panX, velocityX, clock, panState);
 
+  const onCardOffScreen = useCallback(props.onCardOffScreen, []);
```

Comment:
Seems a bit weird that I need to wrap this in useCallback, when it has no dependencies.
Does this make sense to you?

Reply:
This isn't great because any implicit dependencies the passed in prop has won't be respected. Why do you need the useCallback?

Reply:
See https://github.com/ojanvafai/tskod/pull/27/files/fc40273e03697d863bec1be502fb37433c3142d7#r570207423 .

Without it, React is worried that the function's dependencies changing won't cause the useEffect to re-evaluate.